lambda-functions/notification-invoke-service/src/main/java/com/paladincloud/SendNotification.java

```diff
@@ -40,6 +40,26 @@ public String handleRequest(Map<String,Object> event, Context context)
                 System.out.println("bodyJson--"+bodyJson.toString());
                 payloadString = bodyJson.getAsJsonObject().get("Sns").getAsJsonObject().get("Message").getAsString();
             }
+
+            Object payloadObj = gson.fromJson(payloadString,Object.class);
+            if(payloadObj instanceof List){
+                List<Object> notificationRequestList = gson.fromJson(payloadString,List.class);
```

Comment:
@raviteja-mandala @santhosh-challa this line is an error. when sending a object (not an array of events), the code falls through the first if and considers the object to be a list as well